src/main/java/uk/gov/hmcts/reform/managecase/repository/DefaultDataStoreRepository.java

```diff
@@ -109,13 +117,32 @@ public void removeCaseUserRoles(List<CaseUserRole> caseUserRoles, String organis
         dataStoreApi.removeCaseUserRoles(new CaseUserRolesRequest(caseUsers));
     }
 
+    @Override
+    public CaseUpdateViewEvent getStartEventTrigger(String caseId, String eventId) {
+        String userAuthToken = getUserAuthToken();
+        return dataStoreApi.getStartEventTrigger(userAuthToken, caseId, eventId);
+    }
+
+    @Override
+    public StartEventResource getExternalStartEventTrigger(String caseId, String eventId) {
+        String userAuthToken = getUserAuthToken();
+        return dataStoreApi.getExternalStartEventTrigger(userAuthToken, caseId, eventId);
+    }
+
+    @Override
+    public CaseDetails submitEventForCaseOnly(String caseId, CaseDataContent caseDataContent) {
```

Comment:
Can this be renamed to remove the word "Only"?

NB: The previous `submitEventForCase` has be renamed as part of code review changes for ACA-68 in #150.